products/opensuse/main.pm

```diff
@@ -337,6 +337,9 @@ elsif (get_var('GNUHEALTH')) {
     loadtest 'gnuhealth/gnuhealth_client_preconfigure';
     loadtest 'gnuhealth/gnuhealth_client_first_time';
 }
+elsif (get_var('TOOLKITS')) {
+    load_toolkit_tests;
```

Comment:
could you try to schedule in load_common_opensuse_sle_tests instead please?

Reply:
That doesn't seem to work because all the default tests are automatically scheduled before.